variables.tf

```diff
@@ -1,9 +1,33 @@
-##############################################################################
+########################################################################################################################
 # Input Variables
-##############################################################################
+########################################################################################################################
 
-#variable "my_variable" {
-#  type        = string
-#  description = "A description of my variable"
-#  default     = "default_value"
-#}
+variable "enterprise_crn" {
+  type        = string
+  description = "The CRN of the parent Enterprise account to use."
+}
+
+variable "enterprise_primary_contact_iam_id" {
+  type        = string
+  description = "The IAM id of the parent Enterprise account owner."
+}
+
+variable "enterprise_json_input" {
+  description = "List of account groups and account names"
+  type = map(list(object({
+    name         = string
+    owner_iam_id = optional(string, "root")
```

Comment:
I don't understand why `name` and `owner_iam_id` are listed here. They already exist in the `accounts` and `account_groups` options below. When I actually try to set values for them, it fails with:
```
│ Error: Invalid value for input variable
│ 
│   on main.tf line 11, in module "enterprise":
│   11:   enterprise_json_input             = {
│   12:     name = "test"
│   13:     owner_iam_id = "test"
│   14:     account_groups = [{
│   15:       name = "${var.prefix}_account_group",
│   16:       accounts = [{ name = "${var.prefix}_account" }]
│   17:     }]
│   18:   }
│ 
│ The given value is not suitable for module.enterprise.var.enterprise_json_input declared at ../../variables.tf:15,1-33: element "owner_iam_id": list of object required.
```

Reply:
fixed it